DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: admiral
 Type: Package
 Title: ADaM in R Asset Library
-Version: 0.12.0
+Version: 1.0.0
```

Comment:
@ddsjoberg how do we approach this with `usethis` and having .000999 to symbolize a dev version?

Reply:
some advice is coming from @ddsjoberg  on how to handle this soon.  perhaps it can just be done in this PR - i.e. updates to the PR Template